tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/createcsoaccountpayload/CsoPayload.java

```diff
@@ -2,20 +2,32 @@
 
 public class CsoPayload {
 
+    private String universalId;
     private String firstName;
     private String lastName;
     private String middleName;
     private String email;
     Accounts[] accounts;
 
-    public CsoPayload(String firstName, String lastName, String middleName, String email, Accounts[] accounts) {
+    public CsoPayload(String universalId, String firstName, String lastName,
+                      String middleName, String email, Accounts[] accounts) {
+
+        this.universalId = universalId;
```

Comment:
maybe change this to uid? the 'l' with the "I" back to back is panic inducing. any thoughts on this anyone else?

Reply:
This is a pojo class to create the payload so the fields need to match the json object. Thanks.

Reply:
we use `universalId` on fe as well so im ok with it :)